tests/test_structures/test_det3d_data_sample.py

```diff
@@ -152,3 +152,10 @@ def test_deleter(self):
         assert 'pred_pts_seg' in det3d_data_sample
         del det3d_data_sample.pred_pts_seg
         assert 'pred_pts_seg' not in det3d_data_sample
+
+        eval_ann_info = dict(
```

Comment:
Please remove these lines about `eval_ann_info`

Reply:
Thanks for your reminder. It has been removed.